app/services/lyft_service.rb

```diff
@@ -37,6 +37,24 @@ def call_ride(origin, destination)
     @ride_id = json_response[:ride_id]
   end
 
+  def get_estimate(origin, destination)
+    conn = Faraday.new(url: 'https://api.lyft.com') do |faraday|
+      faraday.adapter Faraday.default_adapter
+    end
+    response = conn.get('/v1/cost') do |request|
+      request.headers['Authorization'] = "Bearer #{@user_token}"
+      request.params = {
+                          start_lat: origin[:lat],
+                          start_lng: origin[:lng],
+                          end_lat: destination[:lat],
+                          end_lng: destination[:lng]
+                        }
+    end
+    min_price = JSON.parse(response.body)['cost_estimates'][2]['estimated_cost_cents_min']
```

Comment:
Does the Lyft API allow us to get the type of Lyft as part of the request rather than directly accessing it by index? If not, I'm totally fine with the way we have it. I was just curious if you ran across anything like that in the Lyft docu?